components/isceobj/Alos2Proc/runIonCorrect.py

```diff
@@ -0,0 +1,155 @@
+#
+# Author: Cunren Liang
+# Copyright 2015-present, NASA-JPL/Caltech
+#
```

Comment:
No need to add this. Authorship info is retained by git.

Reply:
removed. I think author info in the file might still be necessary as ISCE has a lot of codes from other sources - we are not the only authors.